### PR TITLE
Reconciler will identify state mismatches between linked gh/jira issues and print to console 

### PR DESCRIPTION
**Pull request by grokspawn**
example output:
image

resolves
- #15 
- #16 
- #17 
- #19

### CODE REVIEW

File: cmd/root/reconcile.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+			err = config.Read()
+			if err != nil {
+				return err
+			}
+
+			jql := ""
+			if config.JiraProject == "" {
+				return fmt.Errorf("must specify jira project")
+			}
+			jql = "project=" + config.JiraProject
```
Comment (oceanc80) [inline cmd/root/reconcile.go:46]: nit: may be cleaner to use fmt.printf() here

**Review by oceanc80 — APPROVED** [review]

File: cmd/root/reconcile.go (hunk 1 above)
Comment (grokspawn) [reply cmd/root/reconcile.go:46]: /hold for cleanup

File: internal/reconcile/reconcile.go [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+}
+
+func splitIssueRef(ref string) (string, int, error) {
+	// split the ref into project (owner/repo), and issue number
+	s := strings.Split(ref, "/")
+	if len(s) <= 4 {
+		return "", 0, fmt.Errorf("unable to extract issue attributes from URL: %v", ref)
+	}
+
+	// URLs to this point have been manicured to the point that they follow the general schema
```
Comment (grokspawn) [inline internal/reconcile/reconcile.go:121]:
```suggestion
	// URLs to this point have been manicured to follow the general schema
```

File: cmd/root/reconcile.go (hunk 1 above)
Comment (grokspawn) [reply cmd/root/reconcile.go:46]: No concern about [cmd/list/list.go](https://github.com/oceanc80/gh2jira/blob/main/cmd/jira/list/list.go#L57-L68)?

Comment (grokspawn) [reply cmd/root/reconcile.go]: (can resolve that in a future PR if there is a concern.. it's not part of this PR currently and this change is made already)

**Pull request merged**